CanDIG/CanDIGv2
# Sonchau/katsu update

What's new:
- This PR updates katsu tests to use v2 and fix the broken test integration

How to test:
- Make sure submodules katsu on `develop` and OPA on `main` and up-to-date
```
make clean-all
make build-all
make init-conda
conda activate candig
make build-all
make test-integration
```
